audit-triplestore/src/main/java/org/fcrepo/camel/audit/triplestore/AuditSparqlProcessor.java

```diff
@@ -62,7 +61,9 @@
      */
     public void process(final Exchange exchange) throws Exception {
         final Message in = exchange.getIn();
-        final UriRef eventURI = new UriRef("XXX");
+        final String eventURIBase = (String) exchange.getProperty("eventURI.base");
```

Comment:
I will leave the question of `property` vs. `header` up to you...
We can certainly leave `eventURI.base` as it is; however, I notice an existing convention along the lines of:
- `triplestore.baseUrl`

Close, but different.